bungee/src/main/java/com/github/games647/fastlogin/bungee/FastLoginBungee.java

```diff
@@ -121,6 +122,13 @@ private void registerHook() {
             BungeeCordAuthenticatorHook hook = new BungeeCordAuthenticatorHook(BungeeCordAuthenticatorBungee, logger);
             core.setAuthPluginHook(hook);
         }
+        Plugin SodionAuth = getProxy().getPluginManager().getPlugin("SodionAuth");
```

Comment:
Could you make this a loop? Explicitly listening the hook process seems like a lot of repetition.